xml/enfinity/6.5/core/processchain.xsd

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+    targetNamespace="https://intershop.github.io/xml/enfinity/6.5/core/processchain"
```

Comment:
the targetNamespace must be not changed.

Reply:
I would prefer to create a new version for processchain.xsd in a new location, which contains the new state.